concrete-csprng/Cargo.toml

```diff
@@ -20,13 +20,18 @@ rand = "0.8.3"
 criterion = "0.3"
 
 [features]
-slow = []
-multithread = ["rayon"]
+parallel = ["rayon"]
+seeder_rdseed = []
+seeder_linux = []
+generator_aesni = []
+generator_soft = []
 
 [[bench]]
 name = "benchmark"
 harness = false
+required-features = ["seeder_rdseed", "generator_aesni"]
 
 [[bin]]
-name = "generate_random"
-path = "src/generate_random.rs"
+name = "generate"
+path = "src/main.rs"
+required-features = ["seeder_rdseed", "generator_aesni"]
```

Comment:
For this profile, the `required-features` thing work. We used this `generate_random` program to verify that our implementation was cryptographically secure, using some statistical tool. We used the `aesni` generator because it was faster !

Reply:
ok thanks!